version/1/4/it/code_of_conduct.md

```diff
@@ -0,0 +1,47 @@
+# Patto di Condotta dei Collaboratori
```

Comment:
Actually, we may translate in two different ways: `Codice di Condotta Convenuto del Collaboratore` or `Patto di Condotta dei Collaboratori`. I would prefer `Patto` because, in one word, we can say  `Codice [...] Convenuto`, which literally translates `Covenant Code`.
The difference between this and the choice done by [Berlin Code of Conduct](http://berlincodeofconduct.org/) is related to the fact that the Berlin Code of Conduct doesn't have `Covenant `on its name.

I believe we should choose together.